executor/executor.cc

```diff
@@ -1146,12 +1146,21 @@ void copyin_int(char* addr, uint64 val, uint64 bf, uint64 bf_off, uint64 bf_len)
 		return;
 	}
 	T x = swap(*(T*)addr, sizeof(T), bf);
-	x = (x & ~BITMASK(bf_off, bf_len)) | ((val << bf_off) & BITMASK(bf_off, bf_len));
+	debug("copyin_int<%d>: old x=0x%llx\n", (int)sizeof(T), (uint64)x);
```

Comment:
Please use debug_verbose.
The debug output can be enabled in execprog and supposed to be relatively high-level. Mostly syscalls executed, return values, timing.
debug_verbose is for syzkaller developers and includes more details.